src/frontend/src/lib/components/auth/AuthGuard.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+	import LandingPage from '$lib/components/auth/LandingPage.svelte';
+	import { authNotSignedIn } from '$lib/derived/auth.derived';
+</script>
+
+{#if $authNotSignedIn}
+	<LandingPage />
+{:else}
+	<div transition:fade>
```

Comment:
You may just need `in` here because on logout we reload the all page anyway

Reply:
Right! Done!